cmd/node/config/prefs.toml

```diff
@@ -50,10 +50,26 @@
    #    { File = "external.toml", Path = "ElasticSearchConnector.Enabled", Value = "true" }
    #]
 
+# BlockProcessingCutoff can be used to stop processing blocks at a certain round, nonce or epoch.
+# This can be useful for snapshotting different stuff and also for debugging purposes.
+# It can only be enabled by using the `--block-processing-cutoff` CLI flag when starting the node
+[BlockProcessingCutoff]
+   # Mode represents the cutoff mode. possible values: "pause" or "processError".
+   # "pause" mode will halt the processing at the block with the given coordinates. Useful for snapshots/analytics
```

Comment:
instead of "pause" maybe we should put "stop" 
Also, "process-error" should technically be "process-error-and-retry"
Not having a very strong opinion

Reply:
I think the current names are fine. 
Even though there isn't a way to 'continue' after 'pause', speaking of the action, it's rather a 'pause' than a 'stop'. Stop makes me think about closing the node.
Also, `process-error` should be enough